Update popup.html

srj0407 opened the pull request:
Asking a question about popups, is this going to be the primary popup menu or is this going to be a nest popup within the main just pertaining to themes? Message me about it!

srj0407 committed:
Update popup.html

Asking a question about popups, is this going to be the primary popup menu or is this going to be a nest popup within the main just pertaining to themes? Message me about it!

CameronDilworth commented:
I personally think we should have a navigation menu, and any popups appear on top of the overlay as a separate popup.

Jacob-Porter committed:
commit to fix stuff

was using other subgroup repo on accident

Jacob-Porter committed:
c/p main -> Subgroup 2

:)

Jacob-Porter committed:
typo fix for syllabus css

:)

Jacob-Porter committed:
css edits

Jacob-Porter committed:
improving darkmode - more html elements and classes affected

progress - still need to make adjustments to colors and other items on other pages

Jacob-Porter committed:
more additions - edits to js

added try-catches in js
couple more darkmode additions to cover more pages

Jacob-Porter committed:
Update custom.css

removed unnecessary !important tags (might add back later...)

Jacob-Porter committed:
edits to css and js

I understand css now - JS and CSS will be updated soon...

Jacob-Porter committed:
more additions to css

Jacob-Porter committed:
Update custom.css

Jacob-Porter committed:
Update custom.css

Jacob-Porter committed:
updated JS and CSS

Completed more pages
Now maintains 'theme' on reload, new window, etc. in 'localStorage'

Jacob-Porter committed:
Update custom.css

announcements page finished

Jacob-Porter committed:
Update custom.css

more more more...

Jacob-Porter committed:
Update custom.css

Jacob-Porter committed:
Update custom.css

/courses page

Jacob-Porter committed:
Update custom.css

calendar

Jacob-Porter committed:
css completed...

Jacob-Porter committed:
Merge branch 'main' into Subgroup2

Jacob-Porter committed:
Merge branch 'main' into Subgroup2

Jacob-Porter committed:
validation test for Grades Feature (Kai's feature)

Jacob-Porter committed:
custom theme progress

now editing css vars with JS for darkmode - working on creating and applying custom themes

Jacob-Porter committed:
minor adjustments

no real changes

Jacob-Porter committed:
good progress on custom themes

minor bug changes to follow...

Jacob-Porter committed:
custom themes done

i think...

also added some more CSS stuff

Jacob-Porter committed:
made popup look nicer, minor changes to custom themes

Jacob-Porter committed:
update input values of selected theme

Jacob-Porter committed:
Update popup.html

Jacob-Porter committed:
update theme when custom theme applied and selected

Jacob-Porter committed:
Update grademath.test.js

added negative tests
